lib/galaxy/datatypes/binary.py

```diff
@@ -736,26 +736,18 @@ def display_peek(self, dataset):
             return "Binary HDF5 file (%s)" % (nice_size(dataset.get_size()))
 
 
-class Trr(Binary):
+class GmxBinary(Binary):
     """
-    Class describing an trr file from the GROMACS suite
-
-    >>> from galaxy.datatypes.sniff import get_test_fname
-    >>> fname = get_test_fname('em.trr')
-    >>> Trr().sniff(fname)
-    True
-    >>> fname = get_test_fname('interval.interval')
-    >>> Trr().sniff(fname)
-    False
+    Base class for GROMACS binary files - xtc, trr, cpt
     """
-    file_ext = "trr"
 
-    def __init__(self, **kwd):
+    def __init__(self, magic_number, file_ext, **kwd):
```

Comment:
That's about it, but you can just make `magic_number` a class attribute that the child classes overwrite, and then you don't need any `__init__` at all.